src/stories/LinksSection.stories.jsx

```diff
@@ -0,0 +1,24 @@
+import LinksSection from "../components/LinksSection";
+
+export default {
+  title: "Links Section",
+  component: LinksSection,
+};
+
+export const Default = {
+  args: {
+    className: "mx-10 lg:mx-52 pb-20",
```

Comment:
can we actually remove this, I would like to view the component in Storybook as is without additional styles to see what the default looks like